src/docs/asciidoc/setting_up_clusters.adoc

```diff
@@ -393,35 +393,38 @@ during the cluster join process. Starting with 3.11, members no longer
 perform a password check when a client connects to the cluster.
 
 [[member-user-code-deployment-beta]]
-=== Deploying User Codes on the Member
-
-Hazelcast can dynamically load your custom classes or domain classes from
-a remote class repository, which typically includes <<enabling-lite-members, lite members>>.
-For this purpose Hazelcast offers a distributed  dynamic class loader.
-
-Using this dynamic class loader, you can control the local caching of the classes
-loaded from other members,
-control the classes to be served to other members and create blacklists or whitelists
-of classes and packages. When you enable this feature, you don't need to deploy your
-classes to all cluster members.
-
-The following is the brief working mechanism of the User Code Deployment feature:
-
-. Dynamic class loader first checks the local classes, i.e., your classpath, for your
-custom class. If it is there, Hazelcast does not try to load it from the remote class repository.
-. Then, it checks the cache of classes loaded from the remote class repository (for this,
-caching should have been enabled in your local, see the <<configuring-user-code-deployment,
-Configuring User Code Deployment section>>). If your class is found here, again, Hazelcast
-does not try to load it from the remote class repository.
-. Finally, dynamic class loader checks the remote class repository. If a member in this
-repository returns the class, it means your class is found and to be used. You can also
-put this class into your local class cache as mentioned in the previous step.
+=== Deploying User Code on the Member
+
+Hazelcast can dynamically load your custom classes or domain classes from other
+members. A <<enabling-lite-members, lite member>> can be designated as a _class
+repository_, but any member can _provide_ classes to other members. For this
+purpose Hazelcast offers a distributed dynamic class loader.
+
+The following is a brief working mechanism of the User Code Deployment feature:
+
+. A new dynamic class loader is created to handle each operation.
```

Comment:
what is "each operation" ?

Reply:
I summarized my understanding from a discussion, I couldn't actually find it in code:

![image](https://user-images.githubusercontent.com/9109877/59010357-3aa24980-8831-11e9-90bb-abe096bc2bed.png)